docs/setup.md

```diff
@@ -49,13 +49,42 @@ sudo apt-get install \
     mlton-compiler haskell-stack
 ```
 
+### macOS packages
+
+First install Homebrew from [their website][homebrewwebsite]. Most of the dependencies
+can be installed with
+
+``` brew install git libxml2 ncurses librsvg dtc cmake ninja texlive rysnc python ccache zstd
+```
```

Comment:
the command needs to be on a separate line from the code fence (```)

`haskell-stack` seems to be missing from the list, which brew also has.

Reply:
Should I include `haskell-stack` in the list here, or should I mention in the Haskell section that macOS users can just perform `brew install haskell-stack`?

Reply:
include it in the package list (same as for Linux)